src/panel-app/apps.component.js

```diff
@@ -38,18 +52,46 @@ export default function Apps(props) {
 }
 
 function sortApps(apps) {
-  return [...apps].sort((a, b) => {
-    const nameA = a.name.toUpperCase(); // ignore upper and lowercase
-    const nameB = b.name.toUpperCase(); // ignore upper and lowercase
-    if (nameA < nameB) {
-      return -1;
-    }
-    if (nameA > nameB) {
-      return 1;
-    }
+  const statusNumMap = {
+    MOUNTED: 2
+  };
```

Comment:
this is really nitpicky but did you want to leave this in for potential future use? Or if this is all we want to do at the moment, can we just turn lines 72/73 into ternary statements or something?